src/main/java/org/folio/circulation/resources/AbstractCirculationRulesEngineResource.java

```diff
@@ -277,9 +278,11 @@ private void apply(RoutingContext routingContext) {
     });
   }
 
-  private CompletableFuture<Result<JsonObject>> buildJsonResult(String policyId){
-    return CompletableFuture.completedFuture(succeeded(new JsonObject().put(getPolicyIdKey(),
-      policyId)));
+  private CompletableFuture<Result<JsonObject>> buildJsonResult(Pair<String, Integer> policyId){
+    return CompletableFuture.completedFuture(succeeded(new JsonObject()
+      .put(getPolicyIdKey(), policyId.getKey())
+      .put("lineNumber", policyId.getValue())
```

Comment:
This is a change to the API and should be reflected in updated interface versions in the RAML and Module Descriptor, and JSON schema.